src/challenges.js

```diff
@@ -44,65 +54,115 @@ const limitFunctionCallCount = (cb, n) => {
 // The cuboid object must contain keys for length, width, and height.
 
 // To test your formulas, pass these key/value pairs into your constructor: length: 4, width: 5, and height: 5. When running your logs, you should get Volume: 100 with a Surface Area of 130. 
+function CuboidMaker(values) {
+  this.length = values.length;
+  this.width = values.width;
+  this.height = values.height;
+}
 
-// Use these logs to test your results:
-// console.log(cuboid.volume()); // 100
-// console.log(cuboid.surfaceArea()); // 130
+CuboidMaker.prototype.volume = function () {
+  return `Volume: ${this.length * this.width * this.height}`;
+}
 
-/* ======================== Class Practice ============================ */
+CuboidMaker.prototype.surfaceArea = function () {
+  return `SurfaceArea: ${2 * (this.length * this.width + this.length * this.height + this.width * this.height)}`;
```

Comment:
It doesn't look like you're closing this function fully. I don't see a matching curly bracket from the opening function.